MobiFlight/InputConfig/InputShiftRegisterConfig.cs

```diff
@@ -11,34 +11,34 @@ namespace MobiFlight.InputConfig
     // fundamental capabilities stay in sync with buttons.
     public class InputShiftRegisterConfig : ButtonInputConfig
     {
-        public int pin;
+        public int ExtPin;
 
         public new object Clone()
         {
             InputShiftRegisterConfig clone = new InputShiftRegisterConfig();
             if (onPress != null) clone.onPress = (InputAction)onPress.Clone();
             if (onRelease != null) clone.onRelease = (InputAction)onRelease.Clone();
-            clone.pin = pin;
+            clone.ExtPin = ExtPin;
             return clone;
         }
 
         public new void ReadXml(System.Xml.XmlReader reader)
         {
-            pin = Convert.ToInt32(reader.GetAttribute(pin));
+            ExtPin = Convert.ToInt32(reader.GetAttribute(ExtPin));
```

Comment:
This is a breaking change. Why did you make the change in this class in the first place?